hazelcast/src/main/java/com/hazelcast/map/impl/operation/PartitionWideEntryBackupOperation.java

```diff
@@ -86,7 +87,7 @@ protected void runWithPartitionScanForNative() {
             Object newValue = outComes.poll();
             EntryEventType eventType = (EntryEventType) outComes.poll();
 
-            operator.init(dataKey, oldValue, newValue, null, eventType, null)
+            operator.init(dataKey, oldValue, newValue, null, eventType, null, UNSET)
```

Comment:
How come you don't use the `outComes.add(operator.getEntry().getNewTtl());` above and set the TTL here? Same in `PartitionWideEntryOperation`. 
Is there an EE test with offloadable and setting TTL?

Reply:
This PR is only relevant for offloadable EP. We had to carry ttl data to partition after  execution of ofloaded EP.
What you pointed is only a signature change when initializing an entry. We don't need to register ttls of entries, this is working as is, no change is done here.

And yes we have `HDEntryProcessorTest` for this PR also.

Reply:
I see. Yes, you're right, you fixed the offloadable behaviour. Still, I see an issue here from before. The operator is used to run the EP for all entries but the new TTL values are discarded and instead we simply set `UNSET` for all entries. You can verify it by modifying `EntryProcessorTest#customTtl_with_entryProcessor` to use `instance1Map.executeOnEntries(entryProcessor)` and run `HDEntryProcessorTest`.

You could fix it in another PR but since you've opened this one, it's easier to fix it here.

Reply:
very good catch, i have added a fix for that also.